source/auth/auth.rst

```diff
@@ -87,6 +87,8 @@ mechanism (string)
 mechanism_properties
 	* Includes additional properties for the given mechanism.
 
+Each mechanism requires certain pieces of information to be present in a MongoCredential for authentication to occur. See the individual mechanism definitions in the "MongoCredential Properties" section. All requirements listed for a mechanism must be met for authentication to occur.
```

Comment:
Replace "pieces of information" with "properties"

Reply:
Done